### PR TITLE
validate

### PR DESCRIPTION
**Pull request by enolfc**
# Summary

---

**Related issue :**

### CODE REVIEW

**Comment by gwarf** [comment]
Nice! But it seems shellcheck is not yet very happy with your script :)

**Comment by enolfc** [comment]
It seems to be working: https://github.com/EGI-Federation/fedcloud-catchall-operations/pull/123/checks?check_run_id=2849270445#step:6:33

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -15,7 +15,9 @@ jobs:
 
       # Runs the Super-Linter action
       - name: Run Super-Linter
-        uses: github/super-linter@v4.1.0
+=        uses: github/super-linter@v4
+=
+==
```
Comment (enolfc) [inline .github/workflows/lint.yml:20]:
```suggestion
         uses: github/super-linter@v4
```

File: .github/workflows/lint.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ jobs:
 
       # Runs the Super-Linter action
       - name: Run Super-Linter
-        uses: github/super-linter@v4.1.0
+         uses: github/super-linter@v4
```
Comment (enolfc) [inline .github/workflows/lint.yml:18]:
```suggestion
        uses: github/super-linter@v4
```

**Comment by enolfc** [comment]
@gwarf can you look at this again? I think it should be ready now

**Review by gwarf — APPROVED** [review]
LGTM

**Pull request merged**